tests/unit/api/test_aggregate_asat.py

```diff
@@ -3,34 +3,17 @@
 """
 import pytest
 
-from featurebyte.api.entity import Entity
 from featurebyte.api.feature import Feature
 from tests.util.helper import check_sdk_code_generation, get_node
 
 
-@pytest.fixture
-def entity_col_int():
-    entity = Entity(name="col_int_entity", serving_names=["col_int"])
-    entity.save()
-    return entity
-
-
-@pytest.fixture
-def scd_view_with_entity(snowflake_scd_table, entity_col_int):
-    """
-    Fixture for an SCDView with entity configured
-    """
-    Entity(name="col_text_entity", serving_names=["col_text"]).save()
-    snowflake_scd_table["col_text"].as_entity("col_text_entity")
-    snowflake_scd_table["cust_id"].as_entity("col_int_entity")
-    return snowflake_scd_table.get_view()
```

Comment:
There were a few `scd_view_with_entity` fixtures in different test / conftest files. Cleaning this part up to keep only one.